pkg/utils/configutil/configutil.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2022 TiKV Project Authors.
```

Comment:
```suggestion
// Copyright 2023 TiKV Project Authors.
```

Reply:
fixed it in another PR which was split from this PR. resolved conflicts.